packages/app/src/api/visualization.js

```diff
@@ -10,10 +10,23 @@ const visualizationQuery = {
     },
 }
 
+const visualizationsQuery = {
+    visualization: {
+        resource: 'visualizations',
+        params: ({ params }) => params,
```

Comment:
Somewhat minor but it's better not to expose the implementation details of what and how these params are constructed.  Since this is only used once and the `fields` are well-defined, it seems like they should be static rather than dynamic.  Also, the query should construct the filter param from a list of IDs rather than accepting a raw, pre-constructed filter string.  I.E.

```js
const visualizationsQuery = {
  visualization: {
    resource: 'visualizations',
    params: ({ visualizationIds }) => ({
      filter: `id:in:[${visualizationIds}]`,
      fields: ['id', 'type']
    })
}
```